setup.py

```diff
@@ -21,18 +24,25 @@
 with open('README.rst') as f:
     readme = f.read()
 
+PY3 = (sys.version_info[0] >= 3)
+
+install_requires = [
+    "pyOpenSSL >= 0.13",
+    "requests >= 2.1.0",
+    "service_identity >= 14.0.0",
+    "six"
+]
+
+if PY3:
```

Comment:
Thanks a ton for the feedback, @dstufft .  I'm still having a little trouble processing what we actually need to do about this though :)